.vuepress/components/deferredIframe.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+    <div v-if="loadNotebook">
+        <iframe width="100%" height="796" frameborder="0" :src="src"></iframe>
+    </div>
+    <div v-else="loadNotebook">
+        <!-- Shouldn't ever see this button unless the intersection API isn't available, this is a fallback -->
+        <a class="action-button" href="#" @click.prevent='loadMe'>Click here to load Observable frame</a>
+    </div>
+</template>
+
+<script>
+export default {
+    props: {
+        src: {
+            type: String,
+            required: true
+        }
+    },
+    data() {
+        return {
+            loadNotebook: false
+        };
+    },
+    mounted() {
+        const observer = new IntersectionObserver(this.loadMe, {
+            threshold: 1.0
+        });
+        observer.observe(this.$el);
+    },
+    methods: {
+        loadMe() {
+            this.loadNotebook = true;
+        },
+    }
+};
+</script>
+<style lang="stylus">
+    // TODO: a spinner or something other than a button, and refactor these styles to be generally applicable.
```

Comment:
haha, ok, I see ... sorry should have looked at this before I submitted my comment :)

Reply:
@bgruening :)  Did you even see the button, or did the intersection observer work and load the iframe as soon as it scrolled into view?

Reply:
I never saw the button but a blank field. A spinner during that time would be nice I think - to indicate that something is happening.

Reply:
![grafik](https://user-images.githubusercontent.com/469983/116256090-280dc200-a773-11eb-8c9c-07fc202f36b1.png)

Reply:
But I guess this is not possible with all the iframe magic.

Reply:
@bgruening Yeah, we'd have to add the spinner on the observable side :/  I can put one on the preload placeholder but it goes away almost immediately once observable starts displaying.

Reply:
Is that something that @spond could add?
Not a big deal I hope.